src/components/common.js

```diff
@@ -130,14 +130,15 @@ export const LabeledCheckbox = ({ checked, onChange, disabled, children, ...prop
   ])
 }
 
-export const tooltip = ({ component, position = 'bottom', arrow = 'right', align = 'right', text, ...props }) =>
-  h(StatefulToolTip, {
-    parent: component,
-    position, arrow, align,
-    group: _.uniqueId(), tooltipTimeout: 0,
-    style: { style: { whiteSpace: 'nowrap', transition: 'none' }, arrowStyle: {} },
-    ...props
-  }, text)
+export const tooltip =
+  ({ component, position = 'bottom', arrow = 'right', align = 'right', text, group = _.uniqueId(), ...props }) =>
```

Comment:
Autogenerating this group can be problematic. I ran into an issue where re-rendering would re-generate the ID, causing the tooltip from the previous render to get stuck on screen. We might need to make all callers supply their own ID.